ui/raidboss/data/04-sb/dungeon/ghimlyt_dark.js

```diff
@@ -41,12 +41,16 @@
       response: Responses.stackMarkerOn(),
     },
     {
+      // 00A7 is the orange clockwise indicator. 00A8 is the blue counterclockwise one.
       id: 'Ghimlyt Dark Magitek Slash',
-      netRegex: NetRegexes.startsUsing({ id: '3774', source: 'Mark III-B Magitek Colossus', capture: false }),
-      infoText: (data, _, output) => output.text(),
+      netRegex: NetRegexes.headMarker({ id: ['00A7', '00A8'] }),
+      infoText: (data, matches, output) => {
+        const direction = matches.id === '00A7' ? 'Left' : 'Right';
```

Comment:
@quisquous how would translation work here?

Reply:
I think I'd consider it an antipattern at this point to build up strings incrementally.  It makes translating hard.  In the past we've done things like `{ en: 'Left' }[data.options.displayLang]` etc, but I think output strings is preferred over code-based maps so that users can edit them too.

In most cases (like this one), I would suggest spelling them out individually, e.g. `rotateLeft` and `rotateRight`.

For other cases where you do want to combine, I would suggest adding `rotate`, `left`, and `right` as output strings and then passing `output.right()` or `output.left()` as a parameter to `output.rotate`, e.g. `output.rotate({ direction: output.right() })`.  However, I think that's unnecessary indirection for this case.

Reply:
Thanks for pointing that out, I'll be more careful about that in future!